tests/test_display.py

```diff
@@ -31,3 +31,33 @@ def test_dislay_invalid_produce_output_line_count(capsys):
     assert "Incorrect" in captured.out
     assert counted_newlines == 2
     assert captured.err == ""
+
+
+def test_reminder_produce_output_line_count(capsys):
+    """Check that the help reminder function produces output lines."""
+    display.help_reminder()
+    captured = capsys.readouterr()
+    counted_newlines = captured.out.count("\n")
+    assert "--help" in captured.out
```

Comment:
All of these tests should probably test for a little bit more than just new lines, although probably not much more.